src/tests/JIT/opt/Compares/compares.cs

```diff
@@ -142,7 +142,7 @@ public static void Gt_ushort_consume(ushort a1, ushort a2)
     public static void Ge_uint_consume(uint a1, uint a2)
     {
         // ARM64-FULL-LINE:      cmp {{w[0-9]+}}, {{w[0-9]+}}
-        // ARM64-NEXT-FULL-LINE: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, {{ge|lt}}
+        // ARM64-FULL-LINE-NEXT: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, hs
```

Comment:
```suggestion
        // ARM64-FULL-LINE-NEXT: csel {{w[0-9]+}}, {{w[0-9]+}}, {{w[0-9]+}}, {{hs|lo}}
```

Just to make these test less fragile (e.g. `csel x0, x1, x2, hs` is equivalent to `csel x0, x2, x1, lo` and the JIT may flip it for any reason)